.github/workflows/build.yml

```diff
@@ -17,7 +17,8 @@ jobs:
         pluto-build: ["Release"] # Only Release supported currently
         mlir-build: ["Release"] #, "Debug"] # "RelWithDebInfo"
         polygeist-build: ["Release", "Debug"] #, "Debug"] # "RelWithDebInfo"
-        polymer: ["0", "1"]
+        polymer: ["1"]
+        pluto: ["1", "0"]
```

Comment:
for ease can we just make a new variable isl, or is it not possible to build with both

Reply:
It's either pluto or isl for now because we use some functions from polly, which bundles its own ISL which conflicts with Pluto's ISL which is a different version

Reply:
(using polly's bundled ISL also allows us to get rid of the external deps for polymer)